File: Python/P35.py
from math import sqrt

if __name__ == '__main__' :
    num = int(input("Enter the number : "))

    # First find all 2s
    while num%2 == 0 :
        print(2, end=" ")

        num /= 2

    # Start from 3 and check for further prime nubers
    i = 3
    while i <= sqrt(num) :
        while num % i == 0 :
            print(i, end = " ")

            num /= i

        i += 2

    # Print the factor
    num > 2 and print(int(num))

File: Arithmetic/P35.cpp
#include <iostream>
#include <math.h>

int main(){
    int num;

    std::cout << "Enter the number : ";
    std::cin >> num;

    for(; num % 2 == 0; std::cout << 2 << " ", num /= 2);

    for(int i = 3; i <= sqrt(num); i += 2)
        for(; num % i == 0; std::cout << i << " ", num /= i);

    if(num > 2) std::cout << num;
    return 0;
}